service/src/main/java/org/folio/rest/workflow/controller/WorkflowController.java

```diff
@@ -102,30 +117,15 @@ public JsonNode workflowHistory(
     return workflowEngineService.history(id, tenant, token);
   }
 
-  @GetMapping(value = { "/search", "/search/" }, produces = { MediaType.APPLICATION_JSON_VALUE })
```

Comment:
Makes sense to put these before the conflicting path `/{id}`. The order can also be explicitly set in a SecurityFilterChain.